service/service.go

```diff
@@ -110,43 +94,42 @@ func Create(
 }
 
 // Start should be called to manage the running of the download service
-func (d Download) Start() {
+func (d Download) Start(ctx context.Context) {
 
 	signals := make(chan os.Signal, 1)
 	signal.Notify(signals, syscall.SIGINT, syscall.SIGTERM)
 
 	d.server.HandleOSSignals = false
-	ctx, cancel := context.WithTimeout(context.Background(), d.shutdown)
 
-	// Start Healthcheck and server
 	d.healthCheck.Start(ctx)
-	d.run()
+	d.run(ctx)
 
 	select {
 	case err := <-d.errChan:
```

Comment:
Should this be triggering a shutdown?

Reply:
good spot! I will log the error, but not shut down in this case.

Reply:
Actually, if we don't want to shut down on this event, I can just remove this channel (the error is logged anyway in run func)